### PR TITLE
[#10144] Add remaining example boxes in question section of instructor-help page

### PR DESCRIPTION
**Pull request by Derek-Hardy**
Part of #10144 

**Snapshots**:

- Team contribution question
Screenshot 2020-06-06 at 00 50 23

- Rubric question
rubric

- Rank recipients question
rank-recipient

**Outline of Solution**:
- add `team contribution` example box & data
- add `rubric question` example box & data
- add `rank recipients` example box & data

### CODE REVIEW

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 1]
```diff
@@ -624,7 +633,16 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
...
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+              <tm-instructor-session-result-question-view
+                  [questions]="exampleRubricQuestionQuestions"
+                  [section]="" [sectionType]="InstructorSessionResultSectionType.EITHER"
+                  [session]="exampleFeedbackSession"
+                  [showStatistics]="true" [indicateMissingResponses]="true"
+                  [isDisplayOnly]="true"
+                  [(instructorCommentTableModel)]="exampleInstructorCommentTableModel">
```
Comment (madanalogy) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:643]: Does the model need to be with the curvy brackets or would square brackets suffice?
```suggestion
                  [instructorCommentTableModel]="exampleInstructorCommentTableModel">
```

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 2]
```diff
@@ -534,7 +534,16 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
...
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+              <tm-instructor-session-result-question-view
+                  [questions]="exampleTeamContributionQuestions"
+                  [section]="" [sectionType]="InstructorSessionResultSectionType.EITHER"
+                  [session]="exampleFeedbackSession"
+                  [showStatistics]="true" [indicateMissingResponses]="true"
+                  [isDisplayOnly]="true"
+                  [(instructorCommentTableModel)]="exampleInstructorCommentTableModel">
```
Comment (xpdavid) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:544]: Same here as @madanalogy has said?

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts [hunk 3]
```diff
@@ -796,6 +967,157 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
...
+        questionType: FeedbackQuestionType.RUBRIC,
+      } as FeedbackRubricResponseDetails,
+      instructorComments: [],
+    },
+  ];
+
+  readonly exampleRubricQuestionQuestionTabModel: QuestionTabModel = {
+    question: this.exampleRubricQuestionModel,
+    responses: this.exampleRubricQuestionResponseOutput,
+    statistics: '', // TODO: need JSON data string
```
Comment (xpdavid) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts:1112]: What is this TODO for? For question type except contribution question, the statistics field is designed to be empty

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 4]
```diff
@@ -742,7 +760,16 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
...
           </ng-template>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+              <tm-instructor-session-result-question-view
+                  [questions]="exampleRankRecipientQuestions"
+                  [section]="" [sectionType]="InstructorSessionResultSectionType.EITHER"
+                  [session]="exampleFeedbackSession"
+                  [showStatistics]="true" [indicateMissingResponses]="true"
+                  [isDisplayOnly]="true"
+                  [(instructorCommentTableModel)]="exampleInstructorCommentTableModel">
```
Comment (xpdavid) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:770]: And here.

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 5]
```diff
@@ -180,7 +180,7 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
                   [session]="exampleFeedbackSession"
                   [showStatistics]="true" [indicateMissingResponses]="true"
                   [isDisplayOnly]="true"
-                  [(instructorCommentTableModel)]="exampleInstructorCommentTableModel">
+                  [instructorCommentTableModel]="exampleInstructorCommentTableModel">
```
Comment (xpdavid) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:183]: 👍

**Pull request merged**